src/assisted_test_infra/test_infra/controllers/node_controllers/vsphere_controller.py

```diff
@@ -21,10 +22,16 @@ class VSphereController(NodeController):
     def __init__(self, config: BaseVSphereConfig, cluster_config: BaseClusterConfig):
         super().__init__(config, cluster_config)
         self.cluster_name = cluster_config.cluster_name.get()
+        self.api_vip = cluster_config.api_vip
+        self.ingress_vip = cluster_config.ingress_vip
```

Comment:
Missed that last review. please remove it and return it using `get_ingress_and_api_vips`

Reply:
sure, changed.